lib/gollum/templates/latest_changes.mustache

```diff
@@ -8,20 +8,22 @@
 
 <div id="wiki-history">
 
-<div class="Box flex-auto">
+<ul class="Box flex-auto">
 	{{#versions}}
-		<div class="Box-row Box-row--hover-gray border-top d-flex flex-items-center">
+		<li class="Box-row Box-row--hover-gray border-top d-flex flex-items-center">
 			<span class="float-left col-2" id="user-icons">{{>author_template}}</span>
 			<span class="flex-auto col-1 text-gray-light">{{date}}</span>
 			<span class="flex-auto col-7">{{message}}<br/>
 			{{#files}}
 			  <span class="flex-auto col-2">{{#renamed}}{{renamed}} -> {{/renamed}}<a href="{{link}}">{{file}}</a></span><br/>
 			{{/files}}
 			</span>
-			<span class="pl-4 float-right">[{{id7}}]</span>
-		</div>
+			<span class="pl-4 float-right">
+				<a href="{{base_url}}/gollum/commit/{{id}}" class="btn btn-outline text-mono">{{id7}}</a>
```

Comment:
It would be nice to be able to use the `page_route` helper, which is defined [here](https://github.com/gollum/gollum/blob/master/lib/gollum/views/helpers.rb#L56), but I don't think it works in the mustache templates at the moment. :(

Reply:
We can use it in the corresponding `views/*.rb` to make a new variable and reference it from mustache.

Reply:
That's maybe not a bad idea! @bartkamphorst what do you think about that development pattern? If it makes sense, we could gradually start doing that for all the places in the templates where we are now manually concatenating stuff to `base_path`.